index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kelvin Munoz</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+    <header>
+        <nav><a href="#homepage">Home</a> <a href="#Biopage">Bio</a> <a href="#Experience">Experience</a> <a href="#portfolio">Portfolio</a></nav>
+    </header>
+<main id = "homepage">
+    
+    <p id = 'quote'> "Your work is going to fill a large part of your life, and the only way to be truly satisfied
+        is to do what you believe is great work. And the only way to do great work is to love what you do." - Steve Jobs      
+        </p>
+
+
+    <Bio >
```

Comment:
if you're using custom HTML tags, be careful with naming conventions - see: https://matthewjamestaylor.com/custom-tags